analytics/views/support.py

```diff
@@ -264,18 +262,19 @@ def support(
             approve_realm_sponsorship(realm, acting_user=acting_user)
             context["success_message"] = f"Sponsorship approved for {realm.string_id}"
         elif modify_plan is not None:
+            billing_session = RealmBillingSession(user=None, realm=realm)
```

Comment:
I think these support flows should use the `acting_user` for `RealmBillingSession` so we have audit logs for which support agent made the change. It is only `billing_session.make_end_of_cycle_updates_if_needed` that should pass `user=None`, because it's really just moving forward work the cron job would do to make sure we've modifying the latest state.